src/Neo/NeoSystem.cs

```diff
@@ -208,16 +218,6 @@ public void EnsureStopped(IActorRef actor)
             inbox.Receive(TimeSpan.FromMinutes(5));
         }
 
-        /// <summary>
-        /// Loads an <see cref="IStore"/> at the specified path.
-        /// </summary>
-        /// <param name="path">The path of the storage.</param>
-        /// <returns>The loaded <see cref="IStore"/>.</returns>
-        public IStore LoadStore(string path)
-        {
-            return StoreFactory.GetStore(store.GetType().Name, path);
-        }
-
```

Comment:
why did you remove this? This will break plugins

Reply:
Mark as obsolete

Reply:
https://github.com/neo-project/neo/pull/3126 now check it here, wont mark it as absolete, still using it. My bad.

Reply:
Its an public `API` should be marked as `obsolete`. If you want to remove it.

Reply:
> We can use Factory in modules, it's not a problem, currently only neo use neo library.

No, I have searched, neoxp also use it, not sure any other tools also using it. @cschuchardt88 is right, its public, cant be changed easily.